tests/iotsdk/accesstoken.go

```diff
@@ -207,13 +207,23 @@ func verifyAccessTokenResponse(response thing.AccessTokenResponse, subject strin
 		anvil.DebugLogger.Println(err)
 		return false
 	}
+	type customClaims struct {
+		SubjectName string `json:"subname,omitempty"`
+	}
 	claims := &jwt.Claims{}
-	if err := accessJWT.UnsafeClaimsWithoutVerification(claims); err != nil {
+	custom := &customClaims{}
```

Comment:
Since the `customClaims` is only used once, you could combine the type definition and the variable creation into one statement.
```
claims := struct {
		SubjectName string `json:"subname,omitempty"`
	}{}
```